host/xtest/aes_gcm_counter_overflow_test_decrypt.h

```diff
@@ -0,0 +1,360 @@
+{ TEE_ALG_AES_GCM, TEE_MODE_DECRYPT, TEE_TYPE_AES,
+/* Key */ (const uint8_t []){0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff, }, 16.0,
```

Comment:
- Missing space after `{`
- Why `16.0`? The type of this field (`struct xt_ae_case::key_len`) is `size_t`.

Reply:
fixed